src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java

```diff
@@ -0,0 +1,82 @@
+package team.five.amazonreviewutil.service;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import team.five.amazonreviewutil.entity.Review;
+import team.five.amazonreviewutil.repositiry.ReviewRepository;
+
+@Service
+public class ServiceParsing {
+
+    @Autowired
+    ReviewRepository reviewRepository;
+
+    private static Logger logger = Logger.getLogger(ServiceParsing.class);
+
+    private final int fieldsCount = 10;
+    private int itemsRead;
+    private final String fieldSeparator = ",";
+    private final String charsetName = "UTF-8";
+    Charset charset = Charset.forName(charsetName);
+    private final char lineBreak = '\n';
+    private static final String DATA_FILE_NAME = "D:\\My\\MA\\project\\test2.csv";
+
+    /*public static void main(String[] args) throws IOException {
+        ServiceParsing serviceParsing = new ServiceParsing();
+        serviceParsing.processInputFile(DATA_FILE_NAME);
+    }*/
+
+    public void processInputFile(String dataFileName) throws IOException {
```

Comment:
@TatianaPirch How long it takes to process the file?

Reply:
about 2 minutes